### PR TITLE
Support for pilight 6.0

### PR DESCRIPTION
**Pull request by idserda**
These changes bring support for pilight 6.0. The configuration and API was changed considerably in this version.

### CODE REVIEW

**Comment by teichsta** [comment]
Thanks @idserda will have a look at this soon, Thomas E.-E.

File: bundles/binding/org.openhab.binding.pilight/src/main/java/org/openhab/binding/pilight/internal/PilightBinding.java [hunk 1]
```diff
@@ -247,9 +206,9 @@ private PilightBindingProvider findFirstMatchingBindingProvider(String itemName)
 		return null;
 	}
 	
-	private List<PilightBindingConfig> getConfigs(String instance, String location, String device) {
+	private List<PilightBindingConfig> getConfig(String instance, String device) {
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.pilight/src/main/java/org/openhab/binding/pilight/internal/PilightBinding.java]: why did you chance the name to singular? A list is returned here so the method name should reflect that properly

File: bundles/binding/org.openhab.binding.pilight/src/main/java/org/openhab/binding/pilight/PilightBindingProvider.java [hunk 2]
```diff
@@ -35,9 +35,9 @@
 	 * can be bound to multiple items in openHAB. 
 	 * 
 	 * @param instance Name of the pilight instance 
-	 * @param location Location of the device in pilight
 	 * @param device Name of the device in pilight
 	 * @return The binding configs for the pilight references or empty list if not found
 	 */
-	public List<PilightBindingConfig> getBindingConfigs(String instance, String location, String device);
+	public List<PilightBindingConfig> getBindingConfig(String instance, String device);
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.pilight/src/main/java/org/openhab/binding/pilight/PilightBindingProvider.java]: why did you chance the name to singular? A list is returned here so the method name should reflect that properly

**Comment by teichsta** [comment]
please find my (minor) comments inline. Thanks, Thomas E.-E.

**Comment by idserda** [comment]
Thanks for the review @teichsta, I've made some changes based on your comments.

**Pull request merged**